Dockerfile

```diff
@@ -1,17 +1,12 @@
-FROM golang:alpine as build
-LABEL org.opencontainers.image.source = "https://github.com/containerish/OpenRegistry"
-
+FROM golang:1.17 as build
+ENV CGO_ENABLED=0
 WORKDIR /root/openregistry
-
-COPY Makefile .
-RUN apk add gcc make git curl ca-certificates
-
 COPY . .
-RUN go mod download && CGO_ENABLED=0 go build -o openregistry -ldflags="-w -s" main.go
+RUN go build -o openregistry -ldflags="-w -s" -trimpath main.go
```

Comment:
@pratikbin thanks for removing these un-needed packages. We actually don't need either of these. The build works fine with just Go toolchain and nothin extra should be needed.